docs/standard/library-guidance/breaking-changes.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Breaking changes
+description: Best practice recommendations for navigating breaking changes when creating .NET libraries.
+author: jamesnk
+ms.author: James.NewtonKing
+ms.date: 09/20/2018
+---
+# Breaking changes
+
+It's important for an open-source project to find a balance between stability for existing users and innovation for the future. As developers, we lend towards refactoring and rethinking code until it's perfect, but breaking your existing users has a negative impact, especially for low-level libraries.
+
+## Project types and breaking changes
+
+How a project is used by the .NET community changes the effect of breaking changes on end-user developers.
+
+* **Low and middle-level libraries** like a serializer, HTML parser, DB ORM, or web framework are the most effected by breaking changes.
```

Comment:
effected => affected (effected => they Did The Thing, affected => the Thing Was Done To Them)

Reply:
I need this on a post it note 🐴